docs/resources/ec_deployment.md

```diff
@@ -139,6 +139,13 @@ resource "ec_deployment" "example_observability" {
 }
 ```
 
+It is possible to enable observability without a second deployment, by keeping observability data in the current deployment. To enable this use `self` as `deployment_id`.
```

Comment:
```suggestion
It is possible to enable observability without using a second deployment, by storing the observability data in the current deployment. To enable this, set `deployment_id` to `self`.
```
I might reword this a bit, but it's pretty clear as is so it's just a suggestion.

Reply:
Thanks, I like your suggestion better :+1: